i18n/en/acknowledgements.md

```diff
@@ -1,7 +1,7 @@
 ## Acknowledgements
 In addition to all Boost and OpenSSL contributors, and all coders since time-immemorial who have made this project possible, we would like to acknowledge the following:
 
-- Wei Dai, Jeffrey Walton, and all the developers at Crypto++ for proving an influential, free C++ library of cryptographical library
+- Wei Dai, Jeffrey Walton, and all the developers at Crypto++ for providing an influential and free C++ cryptographical library
```

Comment:
Please try not to mix English changes with French changes (at least not within the same commit).

Reply:
I am sorry, but I don't find how to split those commits. From what I read here https://gist.github.com/patik/b8a9dc5cd356f9f6f980 I cannot squash if I'm not on master. If you explain me how to do, I will gladly do it. Or I can make a new cleaner pull request?

Reply:
I should've said "for next time", sorry for the confusion. You could do a series of rebase/fixup's but it's more trouble than it's worth, so we can leave this as-is.

Reply:
ok, I tried to do it because @erciccione also asked me to split the commits.
For info, for next time, do you prefer one pull request per file, for the french translations ?

Reply:
Separate PR's and commits for separate languages are preferable. You can PR any number of files for any single language (this is also preferable).